packages/react-table/src/demos/Table.md

```diff
@@ -906,85 +906,875 @@ class ColumnManagementAction extends React.Component {
             <DataListItemRow>
               <DataListCheck
                 aria-labelledby="table-column-management-item1"
-                isChecked={this.state.check1}
+                checked={this.state.check1}
                 name="check1"
+                id="check1"
                 onChange={this.handleChange}
               />
+              <label htmlFor="check1">
               <DataListItemCells
                 dataListCells={[
                   <DataListCell id="table-column-management-item1" key="table-column-management-item1">
                     Repositories
                   </DataListCell>
                 ]}
               />
+              </label>
             </DataListItemRow>
           </DataListItem>
           <DataListItem aria-labelledby="table-column-management-item2">
             <DataListItemRow>
               <DataListCheck
                 aria-labelledby="table-column-management-item2"
-                isChecked={this.state.check2}
+                checked={this.state.check2}
                 name="check2"
+                id="check2"
                 onChange={this.handleChange}
               />
+              <label htmlFor="check2">
```

Comment:
While wrapping the entire cell in a label, there are parent > child relationships in the data list row component elements that this will break. For example, `<DataListItemRow>` is a flex layout, and it expects `<DataListItemCells>` to be a child (it generates `.pf-c-data-list__item-content` [here](https://github.com/patternfly/patternfly-react/blob/de2b31168ee79151ffef751fa26953b0a8d6cb3d/packages/react-core/src/components/DataList/DataListItemCells.tsx#L21)), and `.pf-c-data-list__item-content` is set to `flex-grow` to fill the available space in the row. When the label is introduced to wrap `.pf-c-data-list__item-content`, it breaks that flex parent/child layout, so the `.pf-c-data-list__item-content` no longer fills the row.

Here's what it should look like
<img width="542" alt="Screen Shot 2021-02-11 at 11 20 28 AM" src="https://user-images.githubusercontent.com/35148959/107673021-2d22a100-6c5b-11eb-83a8-1b1b600b6ead.png">

And here's what it looks like now
<img width="432" alt="Screen Shot 2021-02-11 at 11 20 13 AM" src="https://user-images.githubusercontent.com/35148959/107673035-31e75500-6c5b-11eb-8727-67a57900cfc2.png">

All just a long winded explanation for why I would move this `<label>` into `<DataListCell>` and wrap the word(s) you want to serve as the label. After updating, you should see that  `.pf-c-data-list__item-content` grows to fill the entire row.

Reply:
Really great catch, I would not have realized that!! And thanks for the explanation too. 🙂  Just updated!